xml/admin_operating_services.xml

```diff
@@ -244,31 +244,78 @@ ceph-rbd-mirror@.service
   </sect2>
  </sect1>
  <sect1 xml:id="ceph-cluster-shutdown">
-  <title>Graceful Shutdown of the Whole &ceph; Cluster</title>
+  <title>Shutdown and Start of the Whole &ceph; Cluster</title>
 
   <para>
    There are occasions when you need to stop all &ceph; related services in the
    cluster in the recommended order, and then be able to simply start them
    again. For example, in case of a planned power outage.
   </para>
 
-  <para>
-   To start the whole &ceph; cluster, run the <command>ceph.startup</command>
-   runner:
-  </para>
-
-<screen>
-&prompt.smaster;salt-run state.orch ceph.startup
-</screen>
-
-  <para>
-   To shut down the whole &ceph; cluster, disable safety measures and run the
-   <command>ceph.shutdown</command> runner:
-  </para>
-
+  <procedure>
+   <title>Shutting Down the Whole &ceph; Cluster</title>
+   <step>
+    <para>
+     Shut down or disconnect any clients accessing the cluster.
+    </para>
+   </step>
+   <step>
+    <para>
+     To prevent CRUSH to automatically rebalance the cluster, set the cluster
```

Comment:
```suggestion
     To prevent CRUSH automatically rebalancing the cluster, set the cluster

Reply:
copy pasta error...but should this not be "prevent CRUSH from automatically..."?

Reply:
@tbazant either or. "from" is nicer :)

Reply:
I've edited it so you can just add the suggestion :D